core/src/main/java/org/mapfish/print/cli/Main.java

```diff
@@ -37,6 +37,16 @@
  * from other languages than Java.
  */
 public final class Main {
+
+    static {
+        String pkgs = System.getProperty("java.protocol.handler.pkgs");
+        String newValue = "org.mapfish.print.url";
+        if (pkgs != null) {
+            newValue = pkgs + "|" + newValue;
+        }
+        System.setProperty("java.protocol.handler.pkgs", newValue);
```

Comment:
All this code could be moved into a static method in org.mapfish.print.url.data.Handler and be called from all 3 locations where this has been copy and pasted.